clamd.go

```diff
@@ -377,14 +377,16 @@ func (c *Client) streamCmd(tc *textproto.Conn, cmd protocol.Command, f io.Reader
 
 	fmt.Fprintf(tc.W, "n%s\n", cmd)
 	b := make([]byte, 4)
-	for {
+	eof := false
+	for !eof {
 		buf := make([]byte, ChunkSize)
 		if n, err = f.Read(buf); err != nil {
 			if err == io.EOF {
 				err = nil
-				break
+				eof = true
+			} else {
+				return
 			}
-			return
```

Comment:
Set eof to true size we know the error is io.EOF and err to nil

Reply:
So you would prefer to use `break` instead of `for !eof`? (Otherwise the break would be unnecessary.)
I am fine with both but slightly prefer the `for !eof` version (since it better transports the intention of the loop, IMO).